src/helpers/find-titles.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const findTitles = (titles) => {
+  const h1List = [];
+  const h2List = [];
+  const h3List = [];
+  titles.forEach(title => {
```

Comment:
Another thing, at this moment you don't know if this element is title or not, I think this should call htmlElement or element instead of title and the same for titles with htmlElements or elements

Reply:
Okay